apps/pesayetu/src/components/HURUmap/IndicatorTitle/Share.js

```diff
@@ -125,7 +137,28 @@ function Share({
               </CopyToClipboard>
             </div>
           ) : (
-            <ShareButton name={social.name} url={url} {...social.props} />
+            <ShareButton
+              name={social.name}
+              url={url}
+              icon={social.icon}
+              {...social.props}
+              sx={{
+                backgroundColor: `${theme.palette.background.default} `,
+                filter: "opacity(0.6)",
+                width: "100%",
+                border: `solid 1px ${theme.palette.background.paper} `,
+                paddingTop: `${theme.typography.pxToRem(5)} `,
+                "&:hover": {
+                  border: "solid 1px #666666 ",
```

Comment:
Correct me if I'm wrong, but wouldn't it be better to get the border colour from a predefined value in the theme's palette for this part 
 
```
 "&:hover": {
                  border: "solid 1px #666666 ",
```